README.md

```diff
@@ -8,24 +8,50 @@ Member details:
 | Maitry Gajiwala | 40234167 | [MaitryG](https://github.com/MaitryG) | Backend Developer and Tester |
-| Naoreen Chowdhury | 40217016 | | Frontend Developer | 
+| Naoreen Chowdhury | 40217016 | [naoreen](https://github.com/naoreen)| Frontend Developer | 
 | Khushali Parmar | 40224984 | | Frontend Developer | 
 
 # Career Services Platform
 
-<h2>1. High-level project definition</h2>
+## 1. High-level project definition
 The objective of this project is to develop a middle fidelity protoype of a comprehensive Career Services Platform that caters to the needs of both job-seeking students and employers. The platform will enable students to create and update their portfolios while providing employers with the ability to post job openings. There will also be an admin who will manage all user profiles, postings and tracking. The aim of the project is to facilitate efficient job hunting and recruitment processes, benefiting both candidates and employers. 
 
 
-<h2>2. Core Features </h2>
-
-<h2>3. Project Management Philosophy</h2>
-
-1. **Project scope**
-<h2>4. User Roles</h2>
+## 2. Core Features
+**1. Tracking and Reminders for Job Applications**
+- Students can apply for job applications
+- Maintain a central database of a student's job applications, interview times, and hiring phases. 
+- Set reminders for follow-ups and get updates on the status of student's application to keep organized. 
+- Integrate easily with widely used email and calendaring services for effective management.
+
+**2. Job postings and Management**
+- Employers can post job vacancies on the platform, providing detailed information about job roles, responsibilities, qualifications, and desired skills.
+- They can manage their job postings, edit or update them as needed, and set application deadlines.
+- They can see candidates who have applied for the job and schedule an interview if they meet the requirements.
+- They can see candidate profile and resume.
+- Notify employeers when a candidate applies to the job postings.
+  
+**3. Resume Builder and Profile Creation** 
+- Students can create their professional resumes using customizable templates, content suggestions, and formatting guidance.
+- They can showcase their skills, experience, education, and achievements.
+- They can upload portfolio samples, certifications, and attach links to social media or personal websites.
+- Employers can create company profile showcasing company values, job postings, company achievements, posts regarding new product launches, etc.
+
+ **4. Networking and Collaboration**
+ - Students can connect with industry professionals, mentors, and like-minded individuals through networking features.
+ - Explore user directories, engage in chat/messaging, and participate in industry-specific forums/groups.
+ - Collaborate on projects, share documents, and receive valuable feedback through collaboration tools.
+   
+## 3. Project Approach
+(Write the technologies used) TBD
```

Comment:
Once Munj finds the necessary technology most feasible for us; please add it here!